test/test_runner/test_parser.cpp

```diff
@@ -422,8 +422,20 @@ void TestParser::openFile() {
     fileStream.open(path);
 }
 
+std::vector<std::string> TestParser::splitString() {
```

Comment:
Change this function name because it's ambiguous. Also it's rather weird seeing a splitString function not taking any input parameter. 

By design, these type of functions should be a static function that not being exposed in header file.